extensions/positron-r/package.json

```diff
@@ -625,8 +624,10 @@
     "split2": "^4.2.0",
     "vscode-languageclient": "^9.0.1",
     "web-tree-sitter": "^0.20.8",
-    "which": "^3.0.0",
-    "winreg": "^1.2.5"
+    "which": "^3.0.0"
+  },
+  "peerDependencies": {
```

Comment:
Very important to list in `peerDependencies`, not `dependencies`. Keeps `@vscode/windows-registry` out of positron-r's `node_modules/` folder.

The way I learned about this was via a failure to create the macOS universal binary:

```
2024-09-24T23:45:58.381Z electron-universal SHA for file Contents/Resources/app/extensions/positron-r/node_modules/@vscode/windows-registry/build/Makefile does not match across builds 020c678ae1751fa320063785cdd9ad5f0d7c2bdb30c8ab6aad5d6fd6d75bf591!=e38eab36a39a33664e5010b141affc29132e82dd5b7cd10867ca0fb6c1e5d354
Error: Expected all non-binary files to have identical SHAs when creating a universal build but "Contents/Resources/app/extensions/positron-r/node_modules/@vscode/windows-registry/build/Makefile" did not
    at makeUniversalApp (/Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/node_modules/vscode-universal-bundler/dist/cjs/index.js:120:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async origMain (/Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/darwin/create-universal-app.js:171:5)
    at async main (/Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/darwin/create-universal-app.js:161:9)
```

That `Makefile` is generated and differs on arm64 vs x86, hence the failure. But the real solution was to prevent the module from being there in the first place.